signing_service/setup.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+import os
+from setuptools import setup
+
+
+def get_version():
+    with open(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
```

Comment:
It would be more readable as

``` python
def get_version():
    path = os.path.join(
        os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
        'RELEASE-VERSION',
    )
    with open(path, 'r') as version_file:
        return version_file.read()
```

Reply:
Personal matter, but ok :) :+1: done

Done in MiddleMan protocol too.